magefile.go

```diff
@@ -115,10 +115,25 @@ func Spec() error {
 func Integration(testName string) error {
```

Comment:
After [this comment](https://github.com/TBD54566975/ssi-service/pull/195/files#r1038413605) this method will change to use the go test runner with an `integration` flag.